src/config.ts

```diff
@@ -278,7 +278,7 @@ export function parseInput(input: string, config: Config, forbidden: Forbidden[]
 
   // remove forbidden words
   const positive = input.split(/,\s*/g).filter((word) => {
-    word = word.replace(/[^a-z0-9]+/g, ' ').trim()
+    word = word.replace(/[.:_<>|\\\(\)\[\]]+/g, ' ').trim()
```

Comment:
also need to add `\s` and other punctuations (quotation marks, etc.)

Reply:
or simply exclude all of ascii characters except a-z 0-9

Reply:
what about this?